ramls/instance.json

```diff
@@ -215,6 +215,25 @@
             "type": "string",
             "description": "Date (year YYYY) of publication, distribution, etc."
           },
+          "publicationYear": {
+            "type": "integer",
+            "description": "Publication year"
+          },
+          "publicationPeriod": {
+            "type": "object",
+            "description": "Publication period",
+            "properties": {
+              "start": {
+                "type": "integer",
+                "description": "Publication start year"
+              },
+              "end": {
+                "type": "integer",
+                "description": "Publication end year"
+              }
+            },
```

Comment:
I'm aware that Magda referred to these ideas in a comment, yet the issue itself hasn't been updated to include them, has it been confirmed that these new properties are needed and that they are definitely only years (and hence an integer)?